tck/features/OrderByAcceptance.feature

```diff
@@ -0,0 +1,171 @@
+    When executing query: UNWIND ['.*', '', ' ', 'one'] AS strings RETURN strings ORDER BY strings
+    Then the result should be, in order:
+      | strings |
+      | ''      |
+      | ' '     |
+      | '.*'    |
+      | 'one'   |
+    And no side effects
+
+  Scenario: ORDER BY DESC should order strings in the expected order
+    When executing query: UNWIND ['.*', '', ' ', 'one'] AS strings RETURN strings ORDER BY strings DESC
+    Then the result should be, in order:
+      | strings |
+      | 'one'   |
+      | '.*'    |
+      | ' '     |
+      | ''      |
+    And no side effects
+
+  Scenario: ORDER BY should order ints in the expected order
+    When executing query: UNWIND [1,3,2] AS ints RETURN ints ORDER BY ints
+    Then the result should be, in order:
+      | ints |
+      | 1    |
+      | 2    |
+      | 3    |
+    And no side effects
+
+  Scenario: ORDER BY DESC should order ints in the expected order
+    When executing query: UNWIND [1,3,2] AS ints RETURN ints ORDER BY ints DESC
+    Then the result should be, in order:
+      | ints |
+      | 3    |
+      | 2    |
+      | 1    |
+    And no side effects
+
+  Scenario: ORDER BY should order floats in the expected order
+    When executing query: UNWIND [1.5,1.3,999.99] AS floats RETURN floats ORDER BY floats
+    Then the result should be, in order:
+      | floats |
+      | 1.3    |
+      | 1.5    |
+      | 999.99 |
+    And no side effects
+
+  Scenario: ORDER BY DESC should order floats in the expected order
+    When executing query: UNWIND [1.5,1.3,999.99] AS floats RETURN floats ORDER BY floats DESC
+    Then the result should be, in order:
+      | floats |
+      | 999.99 |
+      | 1.5    |
+      | 1.3    |
+    And no side effects
+
+  Scenario: Handle ORDER BY with LIMIT 1
+    And having executed: CREATE (s:Person {name: 'Steven'}), (c:Person {name: 'Craig'})
+
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT 1
+    Then the result should be, in order:
+      | name    |
+      | 'Craig' |
+    And no side effects
+
+  Scenario: ORDER BY with LIMIT 0 should not generate errors
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT 0
+    Then the result should be, in order:
+      | name |
+    And no side effects
+
+  Scenario: ORDER BY with negative parameter for LIMIT should not generate errors
+    And parameters are:
+      | limit | -1 |
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT {limit}
+    Then the result should be, in order:
+      | name |
+    And no side effects
+
+  Scenario: ORDER BY with a negative LIMIT should fail with a syntax exception
+    And having executed: CREATE (s:Person {name: 'Steven'}), (c:Person {name: 'Craig'})
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT -1
+    Then a SyntaxError should be raised at compile time: LiteralMustBePositiveInteger
```

Comment:
I'm not convinced on the word 'Literal' in the error detail. Do you envision that naming it like this will increase our ability to use this detail in other instances? If not, I'd prefer something like `ArgumentIntegerNegative` or `NegativeIntegerArgument`.